packages/design-picker/src/components/design-picker-category-filter/index.tsx

```diff
@@ -24,10 +24,26 @@ export function DesignPickerCategoryFilter( {
 			<VisuallyHidden as="h2" id={ `design-picker__category-heading-${ instanceId }` }>
 				{ __( 'Design categories', __i18n_text_domain__ ) }
 			</VisuallyHidden>
+
+			{ /* Shown on smaller displays */ }
+			<select
```

Comment:
Maybe we can re-use `FormSelect` component?

Reply:
Oh cool, I see that component already has a bunch of custom styles!
I'll see how close it is to the mockups. If we don't have to override too many of the styles to make it match Figma then it would make sense to go with `<FormSelect>`

Reply:
One problem is that `<FormSelect>` is in `client/components`, and I don't know if `@automattic/design-picker` can pull in dependencies from there. I'll see how it goes.

Reply:
It produces an error. We've made it so that you can't import non-package code into packages.
https://github.com/Automattic/wp-calypso/blob/b70a3c7e20c56e7d8498e4d56ff6a62146ffa9c1/.eslintrc.js#L55

That makes sense to me. The packages are supposed to have clean dependency lines so they can be imported from anywhere. `<FormSelect>` isn't really a problem, but you can imagine a package trying to pull in a little bit of calypso code, and in the process having to pull in a huge amount of spaghetti-like dependencies.

I think the correct thing to do (but a bigger change) would be to refactor the form controls out of `calypso/components` and into `@automattic/components`

Reply:
Hmmm...another way is to use the `SelectControl` of Gutenberg but the style might be a little different. So I think we can keep the current approach.